pianobar.cfg.example

```diff
@@ -90,3 +90,18 @@ format_msg_debug = [90m%s[0m
 # Example bindings:
 #hk_act_songpausetoggle = media_play_pause
 #hk_act_songnext = media_next_track
+
+
+# events that should fire the event_command (comma separated);
+# current possible events are:
+# userlogin,usergetstations,stationfetchplaylist,songstart,songfinish
+#
+# note, only songstart and songfinish have all the variable substitution support, other events are not well-tested
+#
+# example usage: 
+# subscribed_events = songstart,songfinish
+subscribed_events = songstart,songfinish
+
+# you can pass variable substitution to event_command to get the current songs.
```

Comment:
This deviates significantly from the upstream implementation which sends (significantly more) metadata to the eventcmd via stdin. As written I think a handler would have to implement support for both styles.

Reply:
Thanks for your review. 

3 thoughts come to mind:

#1. Without the availability of `fork()` on Windows, is there a better alternative than CreateProcess? Should we be trying to reproduce sending the data via stdin using CreateProcess? Does Windows applications/scripts that would be launched from event_command have as part of their regular design patterns to be reading from stdin or is that more of a *nix design pattern?

#2. Stepping away from the implementation details, I would be interested in understanding the metadata requirements. Would there be limitations in passing these to the event_command as arguments such as argument length limits? Is it just the issue that all the metadata fields has not been implemented in the variable substitution parser or is there a fundamental issue with implementing it this way that necessitates for this to be implemented via stdin?

#3. How much do we care to maintain compliance with the upstream project? As I worked on this feature, I noted that the codebases have significantly diverged with parts of code and library being different such as the removal of libcurl for WinHTTP library and DirectShow player instead of original player. Is there interest to maintain config file parity between this port and the original?

Reply:
https://github.com/PowerShell/PowerShell/issues/9497
https://stackoverflow.com/questions/6979747/read-stdin-stream-in-a-batch-file

I didn't dive into it too deeply, but powershell and batch files seem to have limitations with reading from stdin. It would be necessary to utilize python or other non-native scripting language to process from stdin.